docs/glossary.md

```diff
@@ -0,0 +1,19 @@
+| channel            | A channel is an image that each pixel only contains its intensity value (i.e. grayscale image). In fluorescence microscopy, each channel corresponds to a range of wavelengths. An RGB image from digital camera consists of 3 channels - red, green, blue (RGB). In napari, an image dimension that is not spatial or temporal can be considered as channels, and is typically represented as individual [image layer](https://napari.org/tutorials/fundamentals/image.html). See also [this wiki page](https://en.wikipedia.org/wiki/Channel_(digital_image)) for specific examples.       | raster band in geoscience                                                   |
+| colormap           | A colormap represents the value of each pixel as a particular color. There are many different types of colormaps available, such as [matplotlib colormap](https://matplotlib.org/stable/tutorials/colors/colormaps.html).                                                                                                                                                                                                                                                                                                                                                                    | look up table, LUT                                                          |
+| hook               | Hook is the place within napari where we promise to call functions created by external developers and installed by the user. Plugin developers can use hooks to extend functionality of napari. See [this wiki page on hooking](https://en.wikipedia.org/wiki/Hooking) for additional details.                                                                                                                                                                                                                                                                                               |                                                                             |
+| hook specification | A function signature that defines the application programming interface (API) that a plugin developer must adhere to when writing the function that napari promises to call somewhere in the napari codebase. See also [napari hook specifications](https://napari.org/plugins/stable/hook_specifications.html).                                                                                                                                                                                                                                                                             |                                                                             |
```

Comment:
To use the local sphinx links/references instead of a raw url, you can do
```suggestion
| hook specification | A function signature that defines the application programming interface (API) that a plugin developer must adhere to when writing the function that napari promises to call somewhere in the napari codebase. See also {ref}`napari hook specifications <hook-specifications-reference>`.                                                                                                                                                                                                                                                                             |                                                                             |
```